src/advanced/bitcoin-anchoring.md

```diff
@@ -263,59 +273,9 @@ The funding UTXO should get enough confirmations before it can be used.
 However, the network does not check the number of confirmations for the
 provided funding transaction; it is the administrators’ duty.
 
-## Changing Validators List
-
-The list of anchoring keys of validators may be changed for several
-reasons:
-
-- periodic key rotation
-- changing the validators list: add/replace/remove some validators
-
-Both procedures require disabling the old anchoring keys and adding the new
-ones.
-Additionally, as the anchoring Bitcoin address is a derivative from the list of
-anchoring public keys, it should be changed accordingly. The pub-keys list
-is stored in the global configuration; it can be updated by out-of-band
-means, for example, using
-[Configuration Update service](configuration-updater.md). The following
-properties should be taken into account:
+## Maintenance
 
-1. New configuration is spread over nodes. At this point it is not active yet.
-2. New configuration has an additional parameter that indicates the height when
-   this configuration should be applied. The height is set by the
-   administrator.
-3. After the indicated height is reached, the new configuration is applied by
-   all validators simultaneously. At this point the list of validators is
-   finally changed.
-
-!!! tip
-    The administrator should make the interval between the moment a new
-    configuration is proposed and the height when it will be applied sufficient
-    for validators to vote for the new configuration.
-
-!!! warning
-    When switching anchoring to a different address, the administrator should
-    make sure that the service has enough funds to create a
-    [transition transaction](#transition-transaction) to the new address.
-    Therefore, the service will continue the old
-    anchoring chain at a new address. Otherwise, the old anchoring chain will be
-    [lost](#recovering-broken-anchoring).  
-
-### Transition Transaction
-
-Anchoring pubkeys define the anchoring BTC-address. In order to
-prolong the anchoring chain at a new address, a new anchoring transaction should
-spend the
-previous anchoring address UTXO and send it to the new anchoring address. This
-transaction should be committed to the Bitcoin blockchain **before** the updated
-list of validators comes into force. Thus the anchoring process is suspended.
-
-- After the latest anchoring transactions is committed to the Bitcoin
-  blockchain, a transitional anchoring transaction proposal is generated. That
-  transaction moves money to the new anchoring address.
-- As the anchoring chain has already been moved to the new anchoring address,
-  Exonum nodes wait until the new validators set comes into force; after that
-  the anchoring process resumes at the new address.
+Fresh and full maintenance guide you can alway find [here][maintenance-guide].
```

Comment:
Grammar: I'd rephrase as "The maintenance guide is here" (or "...can be found here").